Sources/ReleaseServer/ReleaseServer.WebApi/Repositories/ReleaseArtifactRepository.cs

```diff
@@ -16,18 +14,19 @@ namespace ReleaseServer.WebApi.Repositories
 {
     public class FsReleaseArtifactRepository : IReleaseArtifactRepository
     {
-        private readonly string ArtifactRoot;
+        private readonly string ArtifactRoot, BackupRoot;
         private readonly DirectoryInfo ArtifactRootDir;
         private ILogger Logger;
 
         public FsReleaseArtifactRepository(ILogger<FsReleaseArtifactRepository> logger, IConfiguration configuration)
         {
             ArtifactRoot = configuration["ArtifactRootDirectory"];
+            BackupRoot = configuration["BackupRootDirectory"];
```

Comment:
It should be checked if the directories can be written (check permissions) during server startup.
Otherwise this will not be noticed until a request is performed that results in a writing operation.

This constructor will first be called when a request is performed so this check should be done in the startup method.

Reply:
I will create a new issue targeting this as this does not belong to the actual scope of this PR.